src/tests/system/tests/test_ipa.py

```diff
@@ -390,3 +390,119 @@ def test_ipa__check_gssapi_authentication_indicator(client: Client, ipa: IPA):
     time.sleep(3)
     log2 = client.fs.read(client.sssd.logs.pam)
     assert "indicators: 2" in log2, "String `indicators: 2` not found in logs!"
+
+
+@pytest.mark.importance("high")
+@pytest.mark.topology(KnownTopology.IPA)
+def test_ipa__idview_override_attribute(client: Client, ipa: IPA, moduledatadir: str):
+    """
+    :title: Verify an IPA ID view can override a user or group attribute on the client
+    :description:
+        To ensure that an IPA ID view can successfully override a user or group attribute on the client system.
+    :setup:
+        1. Client is configured to use ID view
+        2. Add a user with override attributes
+        3. Add a group with override attributes
+        4. Restart SSSD on client
+    :steps:
+        1. Check IPA user's overridden attributes
+        2. Check IPA group's overridden attributes
+        3. Check IPA users certificate
+    :expectedresults:
+        1. Successfully get the user's overridden attributes'
+        2. Successfully get the group's overridden attributes'
+        3. Sccessfully get the user's certificate
+    :customerscenario: False
+    """
+    ipa.idview("testview1").add(description="This is a new view")
+    ipa.idview("testview1").apply(hosts=[f"{client.host.hostname}"])
+
+    with open(f"{moduledatadir}/certificate") as f:
```

Comment:
This is missing from the setup, where we load the certificate.  The steps don't match the code, the assertion for the certificate happens last, but the lookup happens early.